### PR TITLE
feat(docker): enhance docker run completion

### PR DESCRIPTION
**Pull request by xwjdsh**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**
image

**Additional info:**

### CODE REVIEW

**Review by fedeci — APPROVED** [review]
There are still a lot of missing options, but this PR LGTM. Thank you!

**Comment by xwjdsh** [comment]
> There are still a lot of missing options, but this PR LGTM. Thank you!

@fedeci I wrote a script and added the missing options.

File: src/docker.ts [hunk 1]
```diff
@@ -1293,19 +1293,659 @@ const sharedCommands: Record<string, Fig.Subcommand> = {
...
+        description: "Container IPv4/IPv6 link-local addresses",
+        args: {
+          name: "list",
+        },
+      },
+      {
+        name: "--log-driver",
+        description: "Logging driver for the container",
+        args: {
+          name: "string",
```
Comment (fedeci) [inline src/docker.ts:1681]:
```suggestion
          name: "string",
          suggestions: ["json-file", "syslog", "journald", "gelf", "fluentd", "awslogs", "splunk", "etwlogs", "gcplogs", "none"],
```

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/docker.ts (hunk 1 above)
Comment (fedeci) [reply src/docker.ts:1681]: this will require to rerun the linter

**Review by mschrage — APPROVED** [review]
Thanks so much for adding all of these options to the spec @xwjdsh! Really appreciate your contribution! 🤩

Would you be interested in becoming a codeowner for the docker completion spec?

File: src/docker.ts (hunk 1 above)
Comment (xwjdsh) [reply src/docker.ts:1681]: Done.

**Comment by xwjdsh** [comment]
@mschrage Of course, I'm working on it.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**